views/VideoView.js

```diff
@@ -0,0 +1,17 @@
+var VideoView = Backbone.View.extend({
+  className: $('#mainVideo'),
+
+  template: Handlebars.compile($('#video-template').html()),
+
+
+  initialize: function() {
+    this.listenTo(this.model, 'change', this.render);
```

Comment:
This isn't necessary unless the individual video models change (which they don't seem to).